types/typescript/PerformanceTimer.d.ts

```diff
@@ -13,15 +13,17 @@ export interface PerformanceTimer {
   /**
    * The timer that is being reported.
    */
-  name:
+  name: (
     | "StartupInitialSync"
     | "StartupIncrementalSync"
     | "StartupStorePreload"
     | "StartupStoreReady"
     | "StartupLaunchScreen"
     | "InitialSyncRequest"
     | "InitialSyncParsing"
-    | "NotificationsOpenEvent";
+    | "NotificationsOpenEvent"
+  ) &
+    string;
```

Comment:
strange to me, but I let @novocaine judge it.

Reply:
this is a kind of superflous [intersection type](https://www.typescriptlang.org/docs/handbook/2/objects.html#intersection-types) saying it needs to be in the list of values and be a string. It doesn't change semantics so i think its ok